moon/utils/server/sv_powerups_manager.moon

```diff
@@ -12,18 +12,15 @@ PowerupManager =
         ply.Powerups[powerupId] = CFCPowerups[powerupId](ply)
 
     plyCanGetPowerup: (ply, powerupId) ->
-        isInPvp = ply\GetNWBool("CFC_PvP_Mode", false)
         powerupBase = CFCPowerups[powerupId]
         existingPowerup = ply.Powerups[powerupId]
 
+        canGet = hook.Run "CFC_Powerups_PlyCanGetPowerup", ply, powerupBase, existingPowerup
+        return canGet if canGet ~= nil
```

Comment:
`IsRefreshable` isnt checked if the hook returns something? Perhaps a
```moonscript
return false if existingPowerup and not existingPowerup.IsRefreshable
```
above the hook would be better?

or just a `return false if canGet == false`, and change the default return of this function to `true`

Reply:
Unless ofc this behaviour is intentional